pontoon/base/migrations/0090_auto_20170413_2101.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-04-13 21:01
+from __future__ import unicode_literals
+
+from bulk_update.helper import bulk_update
+from django.db import migrations, models
+
+TRANSLATOR_LOCALES = {
+    u'ar': u'ar',
+    u'bg': u'bg',
+    u'bs-Latn': u'bs',  # CONFLICT?
+    u'ca': u'ca',
+    u'cs': u'cs',
+    u'cy': u'cy',
+    u'da': u'da',
+    u'de': u'de',
+    u'el': u'el',
+    u'en': u'en-GB',
+    u'es': u'es',
+    u'et': u'et',
+    u'fa': u'fa',
+    u'fi': u'fi',
+    u'fr': u'fr',
+    u'he': u'he',
+    u'hi': u'hi-IN',
+    u'hr': u'hr',
+    u'ht': u'ht',
+    u'hu': u'hu',
+    u'id': u'id',
+    u'it': u'it',
+    u'ja': u'ja',
+    u'ko': u'ko',
+    u'lt': u'lt',
+    u'lv': u'lv',
+    u'ms': u'ms',
+    u'mt': u'',  # MISSING?
+    u'mww': u'',  # MISSING?
+    u'nl': u'nl',
+    u'no': u'',  # MISSING?
+    u'otq': u'', # MISSING?
+    u'pl': u'pl',
+    u'pt': u'pt-PT',
+    u'ro': u'ro',
+    u'ru': u'ru',
+    u'sk': u'sk',
+    u'sl': u'sl',
+    u'sr-Cyrl': u'sr',  # CONFLICT?
+    u'sr-Latn': u'sr',  # CONFLICT?
+    u'sv': u'sv-SE',
+    u'th': u'th',
+    u'tlh': u'',  # MISSING?
+    u'tlh-Qaak': u'',  # MISSING?
+    u'tr': u'tr',
+    u'uk': u'uk',
+    u'ur': u'ur',
+    u'vi': u'vi',
+    u'yua': u'',  # MISSING?
+    u'zh-CHS': u'zh-CN',  # CONFLICT?
+    u'zh-CHT': u'zh-CN'  # CONFLICT
```

Comment:
We use Simplified Chinese script:
https://pontoon.mozilla.org/zh-CN/

So use `zh-CHS`.